wled00/FX.h

```diff
@@ -91,6 +91,7 @@
 //#define SEGLEN           strip._segments[strip.getCurrSegmentId()].virtualLength()
 #define SEGCOLOR(x)      strip.segColor(x) /* saves us a few kbytes of code */
 #define SEGPALETTE       strip._currentPalette
+#define SEGPALETTEIDX    strip._currentPaletteIndex
```

Comment:
Wrong.
Current palette is not a `strip` property, it is an effect property and needs to be stored in Segment object.
See explanation below.

Reply:
The idea was to use SEGPALETTE, which resolves to strip._currentPalette, as the palette cache.
Alternative would be to declare a new global variable for this cache, as I did before in the example.
(i thought that SEGPALETTE was still referred to, but not in active use, so wanted to 'reuse' it efficiently)